### ISSUE TITLE
Can't add URL on public IPv4 or IPv6 address

### ISSUE THREAD

**Issue by Ivanjn** [IgorKha/wireguard-mikrotik#12]
Can you add the possibility of DDNS url? MyMikrotikSerial.sn.mynetname.net (IP - Cloud ) would be great!

**Comment by archont00** [comment]
Indeed - for client's config, it would be nice to have the possibility to enter the peer (server) as FQDN rather than IP address only.

**Comment by IgorKha** [comment]
Yes, it's a good idea, but unfortunately I don't have time to implement it right now.
Feel free to contribute
